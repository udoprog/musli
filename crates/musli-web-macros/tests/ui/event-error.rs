musli_web_macros::define! {
    type Broadcast;

    impl Broadcast for Broadcast {
        impl<'a, 'b> Event for String;
    }

    type Broadcast2;

    impl Broadcast for Broadcast2 {
        impl<T> Event for String;
    }
}

fn main() {}