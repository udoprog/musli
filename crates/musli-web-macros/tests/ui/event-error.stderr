error: Only one lifetime parameter is supported for an `Event` impl
 --> tests/ui/event-error.rs:5:18
  |
5 |         impl<'a, 'b> Event for String;
  |                  ^^

error: Only one lifetime parameter is supported for an `Event` impl
  --> tests/ui/event-error.rs:11:14
   |
11 |         impl<T> Event for String;
   |              ^
